spec/swagger_helper.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.configure do |config|
+  # Specify a root folder where Swagger JSON files are generated
+  # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
+  # to ensure that it's configured to serve Swagger from the same folder
+  config.swagger_root = Rails.root.join('swagger').to_s
+
+  # Define one or more Swagger documents and provide global metadata for each one
+  # When you run the 'rswag:specs:swaggerize' rake task, the complete Swagger will
+  # be generated at the provided relative path under swagger_root
+  # By default, the operations defined in spec files are added to the first
+  # document below. You can override this behavior by adding a swagger_doc tag to the
+  # the root example_group in your specs, e.g. describe '...', swagger_doc: 'v2/swagger.json'
+  config.swagger_docs = {
+    'v1/swagger.yaml' => {
+      openapi: '3.0.1',
+      info: {
+        title: 'API V1',
+        version: 'v1'
+      },
```

Comment:
@AndreaM2429 This is [OPTIONAL]: it would be great if you added more detailed information about the API and documentation.

Consider improving the **title:** by adding a much more descriptive one and it would be good to add a **description:** attribute below the "title:" with a summary about this specific API or documentation. 
```
info: {
title: 'more-descriptive-title',
description: 'description-of-the-API-or-the-docs',
version: 'v1',
}
```

I think this would make the UI docs look much more professional.